test/astar.cc

```diff
@@ -1261,6 +1272,7 @@ TEST(Astar, test_complex_restriction_short_path_fake) {
   vs::TravelMode mode;
   auto costs = vs::CostFactory().CreateModeCosting(options, mode);
   ASSERT_TRUE(bool(costs[int(mode)]));
+  set_hierarchy_limits(costs[int(mode)], true);
```

Comment:
some of these tests circumvent using the workers and call the algorithms directly, so set some default hierarchy limits in the costing.